### PR TITLE
fix: update task profiling and step profiling if already exists

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Review by Kelvin-M — COMMENTED** [review]
LGTM ! 
I think "print" should be removed from unit tests

File: backend/api/tests/views/test_views_task_profiling.py [hunk 1]
```diff
@@ -108,6 +108,17 @@ def test_task_profiling_create_success(self):
             {"count": len(expected_result), "next": None, "previous": None, "results": expected_result},
         )
 
+    def test_already_exist_task_profiling(self):
```
Comment (Kelvin-M) [inline backend/api/tests/views/test_views_task_profiling.py:111]: Is this test failing without de serializer modification ?

Comment (ThibaultFy) [reply backend/api/tests/views/test_views_task_profiling.py:111]: Yes, we began by adding the test

**Review by Kelvin-M — APPROVED** [review]
LGTM !

**Pull request merged**